examples/exchange-files-in-browser/public/app.js

```diff
@@ -98,32 +96,34 @@ function getFile () {
 
   $multihashInput.value = ''
 
-  $errors.classList.add('hidden')
-
-  if (!cid) { return console.log('no multihash was inserted') }
-
-  node.files.get(cid, (err, files) => {
-    if (err) { return onError(err) }
+  if (!cid) {
+    return onError('No multihash was inserted.')
+  }
 
-    files.forEach((file) => {
-      if (file.content) {
-        appendFile(file.name, cid, file.size, file.content)
-      }
+  node.files.get(cid)
+    .then((files) => {
+      files.forEach((file) => {
+        if (file.content) {
+          appendFile(file.name, cid, file.size, file.content)
+          onSuccess(`The ${file.name} file was added.`)
+          $emptyRow.style.display = 'none'
+        }
+      })
     })
-  })
+    .catch((error) => onError('The inserted multihash is invalid.'))
```

Comment:
Shouldn't this return whatever error came from `node.files.get(cid)`? This line seems to assume that the error is always that the multihash was wrong

Reply:
Yes, it assumes the error is an invalid multihash. Before it was returning a stack trace, but in an example app like this I don't think that's the best solution.. neither one are to be honest.

Any suggestions?

Reply:
The error handling is being handled by @hugomrdias . And after this work being completed, (there is a first PR for several commands, files is not one of them yet) we may forward the received error, since the stacktrace will only be received when in debug mode. 
But until then, I think we should provide a friendlier message.

Reply:
Cool, until then I can leave a more general message, something along the lines: 

```
An error occurred when fetching the files.
```

What do you guys think?